sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/RewriteDistinctAggregatesSuite.scala

```diff
@@ -63,7 +63,7 @@ class RewriteDistinctAggregatesSuite extends PlanTest {
     val input = testRelation
       .groupBy('a, 'd)(
         countDistinct('e, 'c).as('agg1),
-        CollectSet('b).toAggregateExpression().as('agg2))
+        DummpAgg('b).toAggregateExpression().as('agg2))
```

Comment:
i was thought about removing it. however, `RewriteDistinctAggregates` rule has logic for this case. this test is the only one for testing this logic. so i didn't remove it in the end.